File: ui/visual-component/src/main/java/org/visual/component/layout/VPadding.java
package org.visual.component.layout;

import javafx.geometry.Insets;
import javafx.scene.layout.Pane;

public class VPadding extends Pane {
  public VPadding(double padding) {
    setVisible(false);
    setWidth(0.0);
    setPrefWidth(0.0);
    setMaxWidth(0.0);
    setPadding(new Insets(0.0, 0.0, padding, 0.0));
  }
}

File: ui/visual-model-component/src/main/kotlin/org/visual/model/component/layout/VPadding.kt
package org.visual.model.component.layout

import javafx.geometry.Insets
import javafx.scene.layout.Pane

class VPadding(padding: Double) : Pane() {
  init {
    isVisible = false
    width = 0.0
    prefWidth = 0.0
    maxWidth = 0.0
    setPadding(Insets(0.0, 0.0, padding, 0.0))
  }
}
